cockroachcloud/cloud-api.md

```diff
@@ -548,7 +548,7 @@ Where:
 
 ## List all the SQL users in a cluster
 
-To list the SQL users in a cluster send a `GET` request to the `/v1/clusters/{cluster_id}/sql-users` endpoint. The service account associated with the secret key must have `ADMIN` or `READ` [permission](console-access-management.html#service-accounts) to list the SQL users.
+To list the SQL users in a cluster send a `GET` request to the `/v1/clusters/{cluster_id}/sql-users` endpoint. The service account associated with the secret key must have `ADMIN` or `READ` [permission](managing-access.html#manage-service-accounts) to list the SQL users.
```

Comment:
In the new auth model, this can be done by Cluster Admin or Cluster Developer roles.